GitSearch/Application/RepoInfo/RepoInfoViewController.swift

```diff
@@ -10,9 +10,9 @@ import UIKit
 
 class RepoInfoViewController: UITableViewController {
     
-    private let cellId = "id"
-    var repInfo = [String: String]()
-    var props = [String]()
+    fileprivate let cellId = "id"
```

Comment:
Why `fileprivate` instead of `private`?

Reply:
`cellId` is something that is using in this class only, and as I understand - fileprivate is more restricted than just private.

Reply:
Not quite: https://www.avanderlee.com/swift/fileprivate-private-differences-explained/